wemake_python_styleguide/violations/best_practices.py

```diff
@@ -1941,3 +1943,43 @@ class FloatKeyViolation(ASTViolation):
 
     error_template = 'Found float used as a key'
     code = 449
+
+
+@final
+class MethodDecoratorUsedForFunctionViolation(ASTViolation):
+    """
+    Forbids descriptors in regular functions.
+
+    Forbids using @staticmethod, @classmethod and @property for functions not
+    in class
+
+    Reasoning:
+        Descriptors like @staticmethod, @classmethod and @property do magic
+        only as methods. We would want to warn users if the descriptors are
+        used on regular functions
+
+    Solution:
+        Do not use @staticmethod, @classmethod and @property on regular
+        functions or wrap the functions into a Class
+
+    Example::
+
+        # Correct:
+        Class TestClass(Object):
+            @staticmethod
```

Comment:
Let's use `@property` here as an example. Because we don't like `@staticmethod`s in this project 🙂